client/src/app/tabs/bpmn-shared/util/configure.js

```diff
@@ -43,61 +43,69 @@ export default function configureModeler(
     };
   });
 
-  const middlewares = [
+  function moddleExtensionsMiddleware(options, logWarning) {
 
-    function moddleExtensionsMiddleware(options, logWarning) {
+    var registeredModdleExtensions = getPlugins('bpmn.modeler.moddleExtension');
 
-      const plugins = getPlugins('bpmn.modeler.moddleExtension');
+    if (platformKey) {
+      registeredModdleExtensions = registeredModdleExtensions.concat(getPlugins('bpmn.' + platformKey + '.modeler.moddleExtension'));
+    }
 
-      const moddleExtensions = plugins.reduce((extensions, extension) => {
-        let {
-          name
-        } = extension;
+    const moddleExtensions = registeredModdleExtensions.reduce((extensions, extension) => {
+      let {
+        name
+      } = extension;
 
-        if (typeof name !== 'string') {
-          logWarning('bpmn.modeler.moddleExtension is missing <name> property');
+      if (typeof name !== 'string') {
+        logWarning('A bpmn moddle extension plugin is missing a <name> property');
```

Comment:
Because not all extensions will match the key 'bpmn.modeler.moddleExtension' anymore, so I've just made the naming more generic.

Reply:
I think it's OK as is. It would be great if we could give more guidance, e.g. which plugin is causing a problem, but that would blow up the scope of the PR.